index.html

```diff
@@ -60,6 +61,188 @@ <h4 class="title">I'am a Software Developer</h4>
+            See Project
+          </button>
+        </div>
+        <div class="project-card inactive">
+          <img src="./images/background2.png" alt="Professional Art Printing Data">
+          <h3 class="project-card-heading">
+            Professional Art Printing Data
+          </h3>
+          <p class="project-card-description">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+          </p>
+          <ul class="skills">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button class="see-project">
+            See Project
+          </button>
+        </div>
+        <div class="project-card inactive">
+          <img src="./images/background2.png" alt="Professional Art Printing Data">
+          <h3 class="project-card-heading">
+            Professional Art Printing Data
+          </h3>
+          <p class="project-card-description">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+          </p>
+          <ul class="skills">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button class="see-project">
+            See Project
+          </button>
+        </div>
+        <div class="project-card inactive">
+          <img src="./images/background2.png" alt="Professional Art Printing Data">
+          <h3 class="project-card-heading">
+            Professional Art Printing Data
+          </h3>
+          <p class="project-card-description">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+          </p>
+          <ul class="skills">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button class="see-project">
+            See Project
+          </button>
+        </div>
+      </div>
+    </section>
+    <!-- about me section -->
+    <section id="about-me">
+      <div class="about-me-intro">
+        <h2 id="about-me-title">About me</h2>
+        <p id="about-me-description">
+          Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <button id="get-resume">Get My Resume</button>
+      </div>
+      <hr>
+      <div class="about-skils-section">
+        <!-- languages -->
+        <div class="list">
+          <div class="list-title">
+            <div class="square-icon rotate"></div>
+            <h3>Language</h3>
+          </div>
+          <ul class="list-items">
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>Css</li>
+          </ul>
```

Comment:
<img width="638" alt="2" src="https://user-images.githubusercontent.com/77843050/219607897-a89f3027-d617-463a-9faf-03719c754763.png">

- [ ] You haev almost reached the perfection, but since our eyes can still see the slight difference between the textes, please make the necessary changes to update it, same as the comment above, consider adjusting your fontweight text as it seems on the original desifn.